lib/OpenQA/Task/Git/Clone.pm

```diff
@@ -26,6 +26,12 @@ sub register ($self, $app, @) {
 sub _git_clone_all ($job, $clones) {
     my $app = $job->app;
     my $job_id = $job->id;
+    my $log = $app->log;
+    my $ctx = $log->context("[#$job_id]");
+    if (($app->config->{global}->{scm} // '') ne 'git') {
+        $ctx->info('Git not enabled in config, skipping all clones.');
+        return;
```

Comment:
I think it would be sufficient to avoid the creation of those tasks. Removing this case would also avoid the failing coverage check.

Reply:
ok. at this point I think it should not be possible to reach this anyway, as I think all paths to it are guarded by an earlier check, so it's just a kind of failsafe. I'll adjust it and add a comment.